src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java

```diff
@@ -404,6 +486,29 @@ public SELF containsSequence(double... sequence) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual array contains the given sequence, without any other values between them.
+   * <p>
+   * Example:
+   * <pre><code class='java'> // assertion will pass
+   * assertThat(new double[] { 1.0, 2.0 }).containsSequence(new Double[] {1.0, 2.0});
+   * assertThat(new double[] { 1.0, 2.0, 2.0, 1.0 }).containsSequence(new Double[] {2.0, 1.0});
+   *
+   * // assertion will fail
+   * assertThat(new double[] { 1.0, 1.0, 2.0 }).containsSequence(new Double[] {2.0, 1.0});</code></pre>
+   *
+   * @param sequence the sequence of values to look for.
+   * @return myself assertion object.
+   * @throws AssertionError if the actual array is {@code null}.
+   * @throws AssertionError if the given array is {@code null}.
+   * @throws AssertionError if the actual array does not contain the given sequence.
+   */
+  public SELF containsSequence(Double[] sequence) {
```

Comment:
Please also add
```java
public SELF containsSequence(Double[] sequence, Offset<Double> precision)
```

Reply:
https://github.com/assertj/assertj-core/blob/c0f48d3858e047ed0421a36c435e6f1b5503edcb/src/main/java/org/assertj/core/api/AbstractDoubleArrayAssert.java#L430

It's already there